client/blocks/taxonomy-manager/list-item.jsx

```diff
@@ -84,7 +85,7 @@ class TaxonomyManagerListItem extends Component {
 					<Gridicon
```

Comment:
I really like the EllipsisMenu, I used it in 7e335ac and it really simplified the component code 👍